src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -731,4 +733,20 @@ var ReactDOMFiberComponent = {
 
 };
 
+if (__DEV__) {
+  ReactDOMFiberComponent.getChildAncestorInfo = function(parentAncestorInfo, type) {
```

Comment:
I don't like these forwarding things. We have so much of these in the event system that just adds bloat, add risk for cyclic dependencies and confusion.

Do you have a plan with these or can you just move this to `ReactDOMFiber`?